### DISCUSSION TITLE
next_dynamic_WEBPACK_IMPORTED_MODULE_12__defult(...) is not a function

### DISCUSSION THREAD

**Discussion by dongrongcmx** [vercel/next.js#14380] [Help]
# Bug report

## Describe the bug

在本地运行的时候，页面会报错next_dynamic_WEBPACK_IMPORTED_MODULE_12__defult(...) is not a function，进而页面不断的自动刷新。报错的动态导入模块实际加载没有问题，打包后上线运行也没有问题。但是在我没有操作页面和代码的时候，不断报错并刷新页面，对开发过程产生了影响，请问这个情况有办法解决吗。

![WechatIMG3](https://user-images.githubusercontent.com/22345076/85193316-e2ce6780-b2f9-11ea-84a6-1a8e0dbd31fe.png)

## System information

- OS: macOS
- Browser (if applies) chrome
- Version of Next.js: 8.0.6
- Version of Node.js: 10.15.3

**Comment by jamesmosier** [comment]
Please provide a full reproduction of your issue.
